pixeltable/globals.py

```diff
@@ -20,11 +20,74 @@
 
 _logger = logging.getLogger('pixeltable')
 
-
 def init() -> None:
     """Initializes the Pixeltable environment."""
     _ = Catalog.get()
 
+def _get_or_drop_existing_path(
```

Comment:
If I had to summarize what this is trying to accomplish, I'd probably say something like "handle SchemaObject path collision".

Reply:
Done. Changed the summary.
<!-- Reviewable comment -OElyFFs4QtMuufUOpUM:b-5dnr5w -->
<!-- Sent from Reviewable.io -->